book/scripts.md

```diff
@@ -878,7 +886,7 @@ do that; we just need to re-apply the styles we already have.[^update-styles]
     `link` nodes are created. This is tricky to get right, so I'm
     skipping it here.
 
-Now, whenever the page changes, we can lay it out again by calling
+Now, whenever the page changes, we can update its rendering again by calling
```

Comment:
Doesn't it need to also call draw? I think the browser has a bug in that render happens but not draw in this case.

Reply:
I answered my own question and added a footnote about it.